docs/testing/stencil-testrunner/_category_.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Non-blocking nit - I think the JSON formatting is a little off here. I'm not opposed to a separate PR to update the Prettier config to check for these things in the future, we simply never checked them before because they so rarely change

Reply:
fixed in https://github.com/ionic-team/stencil-site/pull/1344/commits/141f30873c35b5c9f3f2aa59a6c0521ae0908dc2